src/hdf5/osw_2ioda.py

```diff
@@ -151,17 +165,18 @@ def main(args):
     varDict = defaultdict(lambda: DefaultOrderedDict(dict))
     varAttrs = DefaultOrderedDict(lambda: DefaultOrderedDict(dict))
 
+    obsvars_0s.insert(0, obsvars[0])
     # Set coordinates and units of the ObsValues.
-    iodavar = obsvars[0]
-    # set the obs space attributes
-    varDict[iodavar]['valKey'] = iodavar, obsValName
-    varDict[iodavar]['errKey'] = iodavar, obsErrName
-    varDict[iodavar]['qcKey'] = iodavar, obsQcName
-    varAttrs[iodavar, obsValName]['coordinates'] = 'longitude latitude'
-    varAttrs[iodavar, obsErrName]['coordinates'] = 'longitude latitude'
-    varAttrs[iodavar, obsQcName]['coordinates'] = 'longitude latitude'
-    varAttrs[iodavar, obsValName]['units'] = obsvars_units
-    varAttrs[iodavar, obsErrName]['units'] = obsvars_units
+    for iodavar in obsvars_0s:
```

Comment:
this is incorrect and will not set the variable attributes for the 'Windspeed' variable